keras_contrib/applications/densenet.py

```diff
@@ -164,6 +166,114 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16,
     return model
 
 
+def DenseNetFCN(nb_dense_block=5, growth_rate=12, nb_filter=16, nb_layers_per_block=4,
```

Comment:
Original densenet can also be appropriately parameterized with nb_dense_block_layers this is currently lost from #30.